game.js

```diff
@@ -998,22 +1044,35 @@ const _gameUpdate = (timestamp, timeDiff) => {
     crosshairEl.style.visibility = visible ? null : 'hidden';
   }
 
-  const _updateUse = () => {
-    const useAction = localPlayer.getAction('use');
+  const _updateUse = player => {
+    const useAction = player.getAction('use');
     if (useAction) {
       if (useAction.animation === 'pickUpThrow') {
-        const useTime = localPlayer.actionInterpolants.use.get();
+        const useTime = player.actionInterpolants.use.get();
         if (useTime / 1000 >= throwAnimationDuration) {
-          _endUse();
+          _endUse(player);
         }
       } else if (isMouseUp) {
-        _endUse();
+        _endUse(player);
       }
 
+      if(useAction.animationCombo.length > 0) {
+        const animation = getAnimationDuration(useAction.animationCombo[useAction.index]);
+        if(animation.length > 0) {
+          const animationDuration = animation[0].duration;
+          const useTime = player.actionInterpolants.use.get();
+          if (useTime / 1000 >= 0.6) {
```

Comment:
This is a hack and I'm not sure if we want to do it this way, but if we let the whole animation run if `(useTime / 1000 >= animationDuration)`, combat will be quite slow.